src/psfmachine/tpf.py

```diff
@@ -296,18 +299,40 @@ def load_shape_model(self, input=None, plot=False):
         input : string
             Name of the file containing the shape parameters and weights. The file
             has to be FITS format.
+            If None, then previously computed shape model from Kepler's FFI will be
+            download from https://zenodo.org/record/5504503/ and used as default.
         plot : boolean
             Plot or not the mean model.
         """
         # By default we will load PRF model from FFI when this are ok.
-        # for now this function only works when a file is provided
         if input is None:
-            raise NotImplementedError(
-                "Loading default model not implemented. Please provide input file."
+            # raise NotImplementedError(
+            #     "Loading default model not implemented. Please provide input file."
+            # )
+            input = "%s/data/ffi/ch%02i/%s_ffi_shape_model_ch%02i_q%02i.fits" % (
+                PACKAGEDIR,
```

Comment:
I'm going to merge this as is but I do find f strings to be a little easier to read!